akka-http-core/src/main/scala/akka/http/scaladsl/model/Multipart.scala

```diff
@@ -97,9 +96,9 @@ object Multipart {
      */
     def strictParts: immutable.Seq[Multipart.BodyPart.Strict]
 
-    override def toEntity(charset: HttpCharset, boundary: String)(implicit log: LoggingAdapter = NoLogging): HttpEntity.Strict = {
-      val data = BodyPartRenderer.strict(strictParts, boundary, charset.nioCharset, partHeadersSizeHint = 128, log)
-      HttpEntity(mediaType withBoundary boundary withCharset charset, data)
+    override def toEntityWithLog(boundary: String)(implicit log: LoggingAdapter = NoLogging): HttpEntity.Strict = {
```

Comment:
Is this now OK?  The base now has `toEntity` that delegates down to this override, eventually.  Does that satisfy bin-compat?